terraform/environments/digital-prison-reporting/modules/ec2/scripts/bootstrap_apps.sh

```diff
@@ -111,22 +119,22 @@ cat <<EOF > $kubeconfig
-    server: https://DF366E49809688A3B16EEC29707D8C09.gr7.eu-west-2.eks.amazonaws.com
-  name: live.cloud-platform.service.justice.gov.uk
+    certificate-authority-data: $cp_k8s_cert_auth
+    server: $cp_k8s_server
+  name: $cp_k8s_cluster_name
 contexts:
 - context:
-    cluster: live.cloud-platform.service.justice.gov.uk
+    cluster: $cp_k8s_cluster_name
     namespace: dpr-nomis-port-forwarder
     user: nomis-port-forwarder-migrated
-  name: live.cloud-platform.service.justice.gov.uk
-current-context: live.cloud-platform.service.justice.gov.uk
+  name: $cp_k8s_cluster_name
+current-context: $cp_k8s_cluster_context
 kind: Config
 preferences: {}
 users:
 - name: nomis-port-forwarder-migrated
   user:
-    token: eyJhbGciOiJSUzI1NiIsImtpZCI6IlBiS0MzZGN6a1IwbFljNkNOd1dVODY2OXQzLW0tOWRKZ1dpNEdRTG9LOUkifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJkcHItbm9taXMtcG9ydC1mb3J3YXJkZXIiLCJrdWJlcm5ldGVzLmlvL3NlcnZpY2VhY2NvdW50L3NlY3JldC5uYW1lIjoibm9taXMtcG9ydC1mb3J3YXJkZXItbWlncmF0ZWQtdG9rZW4tMDEtMDEtMjAwMCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJub21pcy1wb3J0LWZvcndhcmRlci1taWdyYXRlZCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjdjOGIyMTllLTJmNGEtNDI4Yi1iYzdiLWU0N2IxNjkzZGZiZCIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDpkcHItbm9taXMtcG9ydC1mb3J3YXJkZXI6bm9taXMtcG9ydC1mb3J3YXJkZXItbWlncmF0ZWQifQ.R0Y9kVzeTE-Bcd4twHNNWFffGvin1yayjtdeQSjNVmQBD42vazcEwbr8H3F5i2A5wW8j5Tk_IYzD7Zby26zyIiu_DF8DXe-bERQEHFa-A6pyygnGQPdWcDlZSklGAHBmUmQH5rKTqKDVfFW8gpP58h8zP3L518pBW6eGi-3GTeJH1pNL7sAVwAaKEmbHWqMScCir7G22MCFYwlVyUTarueFGLxR0fId4y2eHFCarlSBkKSS48ewC9WGCp2knJRZURwdZmlSGGO2mAT4Lqq06ZrIHwe_LUYc8hsWcVSGxqPwhdrHCmIcmhXKz6NMEwFpR6KNoRBBAHziCgurJQHzdKA
```

Comment:
Just one question. Has this token been rotated so that the one stored in the secrets is different?

Reply:
Answered here:
https://mojdt.slack.com/archives/C04Q49Y9E6T/p1716894225353509?thread_ts=1716885669.724609&cid=C04Q49Y9E6T